### ISSUE TITLE
Asking for photo ID at events

### ISSUE THREAD

**Issue by ciaranmcnulty** [symfony/diversity#37]
The joining instructions of most Symfony events contain something like:

> *Important* - You're required to bring your photo ID documents with you (ID card, passport, driving license...) as we need it to verify your registration and hand you your access badge. 

This is a problem because:

1. In some countries (e.g. UK) there is no mandatory ID and not everyone has a passport or driving license. This can be an economic barrier to participation
2. Many people use names that do not match their documentation for a variety of reasons including gender identity, and would find it distressing if forced to use their official names on the registration / lanyards

I haven't seen this requirement be enforced at events often, so perhaps it could be dropped in favour of some sort of email proof (QR code etc.)

**Comment by zanbaldwin** [comment]
Since registration for SymfonyCon Lisbon has already started, as a minimum there should be some white stickers and a permanent marker available at the registration booth for people who are more comfortable using a name other than the one they registered with (along with a sign advertising this is available to do).

**Comment by annesosensio** [comment]
This instruction is only used to enable us at the registration desk to find easily your conference badge and not ask you to repeat several times your name, we try to save time there and we thought that asking for an official ID document to check your name would be easier. We will never refuse to hand out a conference badge if it doesn't really fit to your ID document. We can reword this sentence to have something less strict or formal if you prefer? 

The idea is to make sure we hand out the good badge to the right person so if your name on your badge doesn't totally correspond with your name on your ID it's fine, as long as we can find your badge easily and that at least your family name matches the one on your ID document it's good for us.

I don't know if I make sense here. WDYT?

**Comment by Nyholm** [comment]
I think it does make sense. Maybe we can "soften" this sentence a little bit to be less formal. @ciaranmcnulty do you got a suggestion?

**Comment by ciaranmcnulty** [comment]
Yeah if it's not a hard requirement maybe something like

> If you can bring photo ID documents with you (ID card, passport, driving license...) it may help us locate your access badge quicker

**Comment by lsmith77** [comment]
added a PR related to this
https://github.com/DutchPHPConference/conference-diversity-and-inclusion/pull/5

**Comment by derrabus** [comment]
If it's just about some piece of paper with the correctly spelled name on it, a printed ticket, invoice of confirmation mail could also do the trick, I guess.

**Comment by lsmith77** [comment]
I guess we can close this ticket?

**Comment by ciaranmcnulty** [comment]
Can we re-open this today?

*Important* - You're required to bring your photo ID documents with you (ID card, passport, driving license...) as we need it to verify your registration and hand you your access badge.
Please make sure your SymfonyConnect account name is your real name (not a username or nickname) and matches with your ID card to make sure that you get your attendee badge without any hitches. Badges are produced according to the name you registered on your account. You do not need to download or print any order confirmation or invoice. (Although you can bring them with you if for some reason you're not sure to be fully registered to the conference.)

**Comment by lsmith77** [comment]
```
lsmith77 (he/him) 3:39 PM
I forgot one ticket .. the one about photo ID https://github.com/symfony/diversity/issues/37
I had a quick chat with Emilie about it ..
I think its just a question of wording
Emilie Lorenzo [10:15 AM]
Hello, 
I am sorry but it is mandatory since we are not native English speakers, we preffer to ask ID. We do not ask specifically for IDs but something with name on it
Lynn 3:40 PM
If the problem is identification, what about a scannable QR code or barcode on your phone to identify?
Michelle 3:40 PM
Yeah :)
Timo Bakx 3:40 PM
At DPC they required printed tickets or tickets on phones and scanned the (QR?) codes that were on the tickets.
lsmith77 (he/him) 3:41 PM
good point
Timo Bakx 3:41 PM
Maybe it’s a bit short notice to implement this now, but it would be really nice to move to something like this next year.
Michelle 3:41 PM
If you didn’t have your ticket with you you could still get in with an ID. But it’d be effort for them.
nicolasgrekas 3:41 PM
On this topic: bringing the confirmation email works too, we tell ppl about that before AFAIK.
Michelle 3:41 PM
So it’s just wording at this point I think process wise it’s ok :)
nicolasgrekas 3:41 PM
I think it's a matter of wording in that email.
lsmith77 (he/him) 3:42 PM
@nicolasgrekas do you need support in wording that email?
nicolasgrekas 3:42 PM
ID and/or confirmation email is a MUST, for security at least.
:+1:
1

Michelle 3:42 PM
Basically any proof you bought the ticket. Id or confirmation email.
Lynn 3:42 PM
Carrying an ID is mandatory in the Netherlands as of age 14
:confused:
1
:point_up:
1

nicolasgrekas 3:43 PM
For the wording, I'll have a look and share with you (you can add a note on the issue as a reminder :slightly_smiling_face: )
Michelle 3:43 PM
I think showing an ID could cause some stress if you don’t identify with the information on it :/
lsmith77 (he/him) 3:44 PM
@nicolasgrekas.. I will assign the ticket to you then :slightly_smiling_face:
```

**Comment by lsmith77** [comment]
this is the current text
```
Please make sure your SymfonyConnect account name is your real name (not a username or nickname) and matches with your ID card to make sure that you get your attendee badge without any hitches. Badges are produced according to the name you registered on your account. You do not need to download or print any order confirmation or invoice. Although you can bring them with you if for some reason you're not sure to be fully registered to the conference or not able to show an ID card.
```

we can proposed a reworded version

note the following:
* ID card can be a passport, drivers license and even a credit card
* to make the process slower and less secure the preference is for an ID card over only bringing a print of the order confirmation
* in the Netherlands adults are legally required to always carry an official ID card (correct @linaori ?)

**Comment by linaori** [comment]
In regards of the official ID: https://www.government.nl/topics/identification-documents/compulsory-identification

**Comment by zanbaldwin** [comment]
That webpage lists identity documents issued by the Dutch government - does it apply to foreign visitors, and if so, how are they meant to show one of those documents?

**Comment by TimoBakx** [comment]
> a Dutch passport or a passport or identity card of a country belonging to the EU or the EEA. Passports of all other countries must contain a valid residence sticker

**Comment by lsmith77** [comment]
just send the following alternative propposal:
```Badges are according to the name you registered on your SymfonyConnect account and we usually verify that against your identification document. This way you do not need to download or print any order confirmation or invoice. To speed up the registration process, please make sure that your SymfonyConnect name matches the name in your identification document.
Alternatively you can use the order confirmation email or invoice (as long as your SymfonyConnect name is on it) instead of an identification document for the check-in.```

**Comment by nicolas-grekas** [comment]
I'm proposing the following, using a more straight to the point style hopefully:

> On arrival, you will be required to present an identification document that matches your name on SymfonyConnect. To speed up the registration process, please prepare either your national ID card (preferred) or a similar official document. Alternatively you can print the order receipt or invoice as long as your SymfonyConnect name is on it.

**Comment by TimoBakx** [comment]
Required and alternatively don't really work well together.

**Comment by nicolas-grekas** [comment]
> Required and alternatively don't really work well together.

they don't apply to the same thing!

**Comment by TimoBakx** [comment]
How about:
> Badges are issued to the name you entered in your SymfonyConnect account. In order for us to find your badge, you are required to present either your ID (if your SymfonyConnect name matches the name on your ID) or your confirmation document (that states the correct SymfonyConnect name).

**Comment by nicolas-grekas** [comment]
We don't need to tell about badges, that's implementation details. I prefer the previous formulation it's more imperative. We need to be I think, that secures crowds.

**Comment by lsmith77** [comment]
The final version should imho also state what the preference is for a speedy registration to ensure Emilie's sanity :)

**Comment by nicolas-grekas** [comment]
Note that "an identification document" doesn't mean "an *official* identification document". Actually, the rest explains we accept two kinds of identification doc: official and invoice.

**Comment by TimoBakx** [comment]
Ah, @nicolas-grekas, I get what you mean now. For me it wasn't clear that the "alternatively" was aimed at the national ID card part and not at the identification document part.

Would this be a bit more clear?

> On arrival, you will be required to present an identification document that matches your name on SymfonyConnect. To speed up the registration process, please prepare your national ID card (preferred) or a similar official document. If your SymfonyConnect name does not match your name on your national ID card, you can print the order receipt or invoice as long as your SymfonyConnect name is on it.

**Comment by ciaranmcnulty** [comment]
That last version shows that we understand your docs might not match 👍 

I'd say for London specifically please remember there's no mandated national identity docs so we could emphasise that we'll take, e.g. a credit card with your name on
